static/bidder-info/artechnology.yaml

```diff
@@ -0,0 +1,2 @@
+endpoint: "https://artechnology-prebid.attekmi.com/pbserver/?seat={{.AccountID}}&token={{.SourceId}}"
```

Comment:
Endpoint is not reachable. Please resolve.
```
curl -i --location --request POST https://artechnology-prebid.attekmi.com/pbserver/
HTTP/1.1 404 Not Found
Server: nginx
Date: Tue, 11 Feb 2025 03:52:50 GMT
Content-Type: text/plain
Content-Length: 0
Connection: keep-alive
```

Reply:
Issue was fixed, now endpoint returning 204 code if request sends without necessary parameters

Reply:
Endpoint is reachable
```
curl -i --location --request POST https://artechnology-prebid.attekmi.com/pbserver/
HTTP/1.1 204 No Content
Server: nginx
Date: Tue, 11 Feb 2025 20:53:17 GMT
Connection: keep-alive
Access-Control-Allow-Origin: *
Access-Control-Allow-Credentials: true
```